Gaijinizer patch/Map080.txt

```diff
@@ -431,94 +446,101 @@ I could not attack me ...#MTLed
 そこまで時間を越えた計画を立てた奴らなら
 事前にこの時代の状況も知れたのではないか？
 > CONTEXT: Map080/events/5/pages/1/162/Dialogue/enemy4
-It is my prediction ...... If they made plans
-beyond time to that. Did not you know the
-situation of this era in advance?#MTLed
+This is just a guess but...
+If those two could make plans for events so far
+in advance, it's not a stretch to imagine they
+had advance knowledge of recent events.#Ron
 > END STRING
 
 > BEGIN STRING
 あの服装だから、私の村の連中も騙されたが…
 
 実際に奴らがいつの時代の悪魔かは、知る由もない。
 > CONTEXT: Map080/events/5/pages/1/167/Dialogue/enemy4
-Because of that dress, the people in my village
-were also deceived...In fact, there is no way
-to know when they were a demon.#MTLed
+Their style of dress fooled the people of my
+village... There's actually no way of knowing
+just what era those demons are from.#Ron
 > END STRING
 
 > BEGIN STRING
 物言いも謎だったしね……
 
 『我』がどうとか……
 > CONTEXT: Map080/events/5/pages/1/171/Dialogue/kazuya
-The story was a mystery too ...
-"I" is how .........#MTLed
+The way they spoke was mysterious too...
+
+Like why they always emphasized the word "I"...?#Ron
 > END STRING
```

Comment:
Note

Reply:
I just had a check and it looks like I translated all the ones I found from "I" to "me" for grammatical reasons.

Reply:
Probably best to swap "I" with "Me" here, then.